WashU-IT-RIS/coldfront-wustl-fork
# Feature/itdev 35291 user readme access acl

Read-only access given to anyone who can access the README.txt in an allocation.

Testing Instructions

Pull ITDEV-35291 and build locally (making sure that the .env file targets storage2-dev)
Launch coldfront (local) and create a two new allocations. (add ad-ris-qa as a rw user for one and add ad-ris-qa as a wo user for the other)
retrieve ad-ris-qa log in information from pass (pass shared-user-accounts/RIS-SVC-QA)
ssh [ad-ris-qa@compute-dev-exec-2.ris.wustl.edu](mailto:ad-ris-qa@compute-dev-exec-2.ris.wustl.edu)
cd /storage2-dev/fs1/allocation_name
vim README.txt then :w inside vim window
verify that "E45: 'readonly' option is set (add ! to override)" displays in the lower left corner of the vim window
